src/main/webapp/app/shared/category-selector/category-selector.component.scss

```diff
@@ -34,7 +34,12 @@ jhi-category-selector {
     }
 }
 
-// Otherwise the x will jump when opening the colorpicker
+/* Otherwise each custom tag will have an additional border */
+.mat-mdc-standard-chip .mdc-evolution-chip__action--primary:before {
```

Comment:
Can you double-check this line? Codacy is reporting an issue here:
![image](https://github.com/ls1intum/Artemis/assets/38403547/baab96e3-53bd-4f11-8693-43ce7f3b6070)

Reply:
Hey, thanks a lot, you are right. I'll fix it right now.